### DISCUSSION TITLE
surveymarks/k-19/

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#27] [Announcements]
# surveymarks/k-19/

http://localhost:4000/surveymarks/k-19/

**Comment by jengalas** [comment]
Posted by Rodney L. Hanscom
July 13, 2021 at 10:45 am
_Imported from WordPress_

Judy and I recovered this mark years ago, perhaps around 2005, but I can’t find data for it on either the NGS site nor “BM Viewer” by Scaredycatfilms. Can’t say I recall submitting a recovery report. Can you tell me where you found the “Official USGS Description” (above), so that I can track it down? Does it have a PID?
Thanks,
Rodney

**Reply by jengalas** [reply]
Posted by Zhanna
July 15, 2021 at 6:18 pm
_Imported from WordPress_

USGS datasheets are not available in digital format, other than the ones I have scanned myself and posted here:

[https://thesurveystation.com/usgs-datasheets/](https://thesurveystation.com/usgs-datasheets/)

You will find the quad containing K 19 here:

[https://geodata.thesurveystation.com/USGS/ME/36-Mount-Desert/Vertical.pdf](https://geodata.thesurveystation.com/USGS/ME/36-Mount-Desert/Vertical.pdf) (it’s on page 9 of the PDF).

This mark is not in the NGS database and therefore doesn’t have a PID.

I would be thrilled if you found the mark! We looked for it twice over the years and had no luck. Do you happen to have any recollections about it, or perhaps some photos? We have found most of the other “K”-series marks in the area, but not that one. Unless the description is way off, we are expecting to need a metal detector to find it as we have searched the correct area (based on the description and coordinates taken from the topo map) pretty thoroughly—there aren’t many exposed ledges or outcrops there.

**Reply by jengalas** [reply]
Posted by Rodney L. Hanscom
July 19, 2021 at 10:18 pm
_Imported from WordPress_

Replied by email. We’re gonna nail this one.
